### ISSUE TITLE
Accessibility info is missing from README

### ISSUE THREAD

**Issue by MelSumner** [patience-tema-baron/ember-keyboard#97]
In order to ensure that well-meaning Ember developers do not override keyboard shortcuts that are already in use by screen readers and other assistive technology, please add a notice about keyboard shortcuts and gestures to the README. 

It could be something like this: 

## Accessibility Considerations
In order to avoid adding keyboard shortcuts to your application that are already in use by assistive technology, please review the existing keyboard shortcuts and gestures available today: https://dequeuniversity.com/screenreaders/. 

Thank you!

**Comment by patience-tema-baron** [comment]
This is really helpful! If you want to make a quick pr w/ that change, I'd be happy to merge.

**Comment by MelSumner** [comment]
Thank you @patience-tema-baron! PR here: https://github.com/patience-tema-baron/ember-keyboard/pull/98

**Comment by patience-tema-baron** [comment]
Thank you so much!!
